actionpack/lib/action_dispatch/routing/mapper.rb

```diff
@@ -64,6 +64,7 @@ def self.build(scope, set, ast, controller, default_action, to, via, formatted,
 
           defaults = (scope[:defaults] || {}).dup
           scope_constraints = scope[:constraints] || {}
+          to ||= options[:to]
```

Comment:
@rafaelfranca of course we can. The only tradeoff is that in case this would be called in more than one place we would have to do that everywhere. Which is probably irrelevant cause it's very unlikely that we will call `Mapping.build` in more places than one. Nevertheless it's a conscious decision we have to take.